File: 自定义头像/自定义头像/ViewController.m
//
//  ViewController.m
//  自定义头像
//
//  Created by BoBo on 17/2/20.
//  Copyright © 2017年 Li Sen. All rights reserved.
//

#import "ViewController.h"
#import "CustomView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    CustomView *cView = [[CustomView alloc] initWithFrame:CGRectMake(0.0, 200.0, 30.0, 30.0)];
    [self.view addSubview:cView];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 支付宝加载动画/支付宝加载动画/ViewController.swift
//
//  ViewController.swift
//  支付宝加载动画
//
//  Created by BoBo on 17/3/14.
//  Copyright © 2017年 Li Sen. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    fileprivate lazy var animationView: AnimationView = {
        let view: AnimationView = AnimationView(frame: CGRect(x: 0.0, y: 64.0, width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.height))
        
        return view
    }()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        view.addSubview(animationView)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

